src/tapis-ui/_common/Progress/Progress.tsx

```diff
@@ -0,0 +1,53 @@
+type ProgressProps = {
+  value: number;
+  color?: string;
+  height?: number;
+  styles?: { [name: string]: string };
+  showProgress?: boolean;
+};
+
+const Progress: React.FC<ProgressProps> = ({
+  value,
+  color,
+  height = 20,
+  styles,
+  showProgress = true,
+}) => {
+  return (
+    <div
```

Comment:
So... it's more difficult to do, but these styles should be applied with a scss module and box constraints for heights, rather than programmatically calculated.